google/cloud/forseti/common/gcp_api/compute.py

```diff
@@ -574,10 +573,9 @@ def list_instances(self, resource, instance_group, region, **kwargs):
             self, resource, verb='listInstances', **kwargs)
 
 
-class _ComputeSubnetworksRepository(
-        repository_mixins.AggregatedListQueryMixin,
-        repository_mixins.ListQueryMixin,
-        _base_repository.GCPRepository):
+class _ComputeSubnetworksRepository(repository_mixins.AggregatedListQueryMixin,
+                                    repository_mixins.ListQueryMixin,
```

Comment:
This indentation is not aligned.

Reply:
Sorry, but I don't see how it's not aligned.

Reply:
Okay, no-ops.